### PR TITLE
Marshal cleanup

### PR DESCRIPTION
**Pull request by NathanTP**
A bit of an overhaul of this benchmark, particularly around the marshal workload descriptions.

- Bump speckle to get updated metrics and build procedures
- New marshal workloads for quick tests in addition to the full suites
- Updated marshal workloads to add post-run-hooks for metric collection and plotting. Also have working host-init stuff to make builds smooth.

### CODE REVIEW

File: handle-results.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+import pathlib
+import sys
+import pandas as pd
+import argparse
+
+# All time measurements are in seconds
+
+# rate suite not supported yet
+# Reference times taken from https://www.spec.org/cpu2017/results/res2017q2/cpu2017-20161026-00003.html
```
Comment (abejgonzalez) [inline handle-results.py:10]: What was the reasoning to compare against an UltraSPARC vs something in-house? Was the assumption that someone will just make better plots out of the data later and this is just a placeholder?

**Comment by abejgonzalez** [comment]
From what I can tell, this seems fine. However, I think the best test will be to use this when testing SonicBOOM for the RTML tapeout that is coming out... so Jerry can probably comment more on that.

File: handle-results.py (hunk 1 above)
Comment (NathanTP) [reply handle-results.py:10]: This is the official spec score baseline as defined by the org. When reporting SPEC numbers, you're supposed to compare to that machine for apples-to-apples comparison.

**Comment by NathanTP** [comment]
Forgot about this PR, whoops. @jerryz123 any objections?

**Review by jerryz123 — APPROVED** [review]

**Comment by jerryz123** [comment]
LGTM

**Pull request merged**